client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/useInvestigatedPersonInfo.ts

```diff
@@ -46,64 +49,66 @@ const useInvestigatedPersonInfo = (): InvestigatedPersonInfoOutcome => {
     };
 
     const confirmExitUnfinishedInvestigation = (epidemiologyNumber: number) => {
-        Swal.fire({
-            icon: 'warning',
-            title: 'האם אתה בטוח שתרצה לצאת מהחקירה ולחזור אליה מאוחר יותר?',
-            showCancelButton: true,
-            cancelButtonText: 'בטל',
-            cancelButtonColor: theme.palette.error.main,
-            confirmButtonColor: theme.palette.primary.main,
-            confirmButtonText: 'כן, המשך',
-            customClass: {
-                title: classes.swalTitle,
+        if(investigationStatus.subStatus === transferedSubStatus && !investigationStatus.statusReason) {
+                alertWarning('שים לב, כדי לצאת מחקירה יש להזין שדה פירוט');
```

Comment:
You must insert all of the other options to the function as well, as in ContactQuestioningInfo line 40

Reply:
fixed